bench/AElf.Benchmark/BlockExecutingTests.cs

```diff
@@ -51,7 +52,7 @@ public async Task IterationSetup()
                     ChainId = chain.Id,
                     Height = chain.BestChainHeight + 1,
                     PreviousBlockHash = chain.BestChainHash,
-                    Time = Timestamp.FromDateTime(DateTime.UtcNow)
+                    Time = DateTimeHelper.Now.ToTimestamp()
```

Comment:
TimestampHelper.Now, I remembered some one has an implement

Reply:
Sure, I will use that one